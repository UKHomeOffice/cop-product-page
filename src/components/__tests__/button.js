import React from "react";
import renderer from "react-test-renderer";
import Button from "../button";

describe("button", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Button/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns the correct button 'text' prop passed in", () => {
    const tree = renderer
      .create(<Button text={"submit"}/>);
    const testInstance = tree.root;
    expect(testInstance.props.text).toBe("submit");
  });
});