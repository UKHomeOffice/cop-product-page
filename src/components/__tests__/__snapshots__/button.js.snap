// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button renders correctly 1`] = `
<button
  className="govuk-button"
  data-module="govuk-button"
  id="a"
  onClick={[Function]}
/>
`;
